### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by rohit-rk**

### CODE REVIEW

**Comment by klahnakoski** [comment]
I see the README has a summary, good.  Be sure to add install instructions (`pip install` should work), and some examples on how to use it.  Some sections of the README.md can be pulled into another doc.

**Comment by rohit-rk** [comment]
Yes, pip installation works, added. And made few updates too, removing parts from the readme. Please review.

File: README.md [hunk 1]
```diff
@@ -1,126 +1,54 @@
...
-Child tables have a `_id` column, plus two others: `_order` so we can reconstruct the original JSON and `_parent` which is used to refer to the immediate parent of the array.
 
-## More Design Docs
+## Docs
 
+* [Json Query Expression](https://github.com/klahnakoski/ActiveData/blob/dev/docs/jx.md)
+* [Nomenclature](https://github.com/mozilla/jx-sqlite/blob/master/docs/Nomenclature.md)
 * [Snowflake](https://github.com/mozilla/jx-sqlite/blob/master/docs/Perspective.md)
 * [JSON in Database](https://github.com/mozilla/jx-sqlite/blob/master/docs/JSON%20in%20Database.md)
+* [The Future](https://github.com/mozilla/jx-sqlite/blob/master/docs/The%20Future.md)
```
Comment (rohit-rk) [inline README.md]: The links might not work for now, these are set to the main repo.

File: README.md [hunk 2]
```diff
@@ -1,126 +1,54 @@
...
+The most interesting, and most important feature is that we query nested object arrays as if they were just another table.  This is important for two reasons:
 
-### Nested Objects
+1. Inner objects `{"a": {"b": 0}}` are a shortcut for nested arrays `{"a": [{"b": 0}]}`, plus
+2. Schemas can be expanded from one-to-one  to one-to-many `{"a": [{"b": 0}, {"b": 1}]}`.
 
-When it comes to nesting objects, a new table will be required 
 
-    {"a": [{"b": 4}, {"b":5}]}
+There are over 200 tests used to confirm the expected behaviour: They test a variety of JSON forms, and the queries that can be performed on them. Most tests are further split into three different output formats ( list, table and cube).
```
Comment (klahnakoski) [inline README.md]: put this under tests

**Comment by klahnakoski** [comment]
I have been reading your updates, and they all look fine. 

For the future, if you have a pull request for doc changes, I will simply merge them with without review.  I see no need to slow down documentation by forcing a review process; Docs are for humans, and humans are delightfully forgiving of documentation problems, and we can fix them at any time.

I have not merged this because of the "no pull" title prefix.

**Pull request merged**